node-operators/networks/collators/requirements.md

```diff
@@ -0,0 +1,147 @@
+
+### 其他Moonkey功能 {: #other-moonkey-features }  
+
+Moonkey也提供一些其他的功能，具体如下所示：
+
+- `-help` – 显示帮助信息
+- `-version` – 显示您所运行的Moonkey版本
+- `-w12` – 生成12个单词的助记词（默认为24个）
+
+还有以下可用选项：
+
+- `-account-index` – 提供要在派生路径中使用的帐户索引
+- `-mnemonic` – 提供助记符
+
+## 绑定数量要求 {: #bonding-requirements }
+
+作为收集人您需要知道两个绑定数量：一个是加入收集人池的绑定数量，另一个是密钥关联的绑定数量。
+
+### 最低收集人绑定数量 {: #minimum-collator-bond }
+
+首先，您需要拥有最低Token质押量（即收集人自身绑定的最低数量）才有资格成为候选收集人。只有一定数量的总质押量（包括收集人自身绑定的数量和委托的质押量）排名靠前的候选收集人才会进入收集人有效集。
+
+=== "Moonbeam"
+    |         变量          |                          值                           |
+    |:-------------------------:|:--------------------------------------------------------:|
+    | 最低自身绑定数量  |     {{ networks.moonbeam.staking.min_can_stk }}枚GLMR     |
+    | 最低总绑定数量 |     {{ networks.moonbeam.staking.min_col_stk }}枚GLMR     |
+    |      有效集上限      | {{ networks.moonbeam.staking.max_candidates }}名收集人 |
+
+=== "Moonriver"
+    |         变量          |                           值                           |
+    |:-------------------------:|:---------------------------------------------------------:|
+    | 最低自身绑定数量  |     {{ networks.moonriver.staking.min_can_stk }}枚MOVR     |
+    | 最低总绑定数量 |     {{ networks.moonriver.staking.min_col_stk }}枚MOVR     |
+    |      有效集上限      | {{ networks.moonriver.staking.max_candidates }}名收集人 |
+
+=== "Moonbase Alpha"
+    |         变量          |                          值                           |
+    |:-------------------------:|:--------------------------------------------------------:|
+    | 最低自身绑定数量  |     {{ networks.moonbase.staking.min_can_stk }}枚DEV      |
+    | 最低总绑定数量 |     {{ networks.moonbase.staking.min_col_stk }}枚DEV      |
+    |      有效集上限      | {{ networks.moonbase.staking.max_candidates }}名收集人 |
+
+
+### 密钥关联绑定数量 {: #key-association-bond }
+
+其次，您将需要为密钥关联绑定数量。当[映射author ID](/node-operators/networks/collators/account-management)（会话密钥）与您的帐户获取区块奖励时，将发送绑定数量，并且该绑定数量是根据的author ID注册的。
+
+=== "Moonbeam"
+    |   变量   |                         值                          |
+    |:------------:|:------------------------------------------------------:|
+    | 最低绑定数量 | {{ networks.moonbeam.staking.collator_map_bond }}枚GLMR |
+
+=== "Moonriver"
+    |   变量   |                          值                          |
+    |:------------:|:-------------------------------------------------------:|
+    | 最低绑定数量 | {{ networks.moonriver.staking.collator_map_bond }}枚MOVR |
+
+=== "Moonbase Alpha"
+    |   变量   |                         值                         |
+    |:------------:|:-----------------------------------------------------:|
+    | 最低绑定数量 | {{ networks.moonbase.staking.collator_map_bond }}枚DEV |
+
+## 收集人问卷调查 {: #collator-questionnaire }
+
+[收集人问卷调查](https://docs.google.com/forms/d/e/1FAIpQLSfjmcXdiOXWtquYlBhdgXBunCKWHadaQCgPuBtzih1fd0W3aA/viewform)旨在评估所有收集人在Moonbase Alpha上状态。因此，在填写此表格之前，您需要先在Moonbase Alpha上运行一个收集人节点。您将需要在表格中提供您的联系方式和一些基本的硬件设备规格。比如：“请提供您的联系方式，便于我们在发现您的节点无法正常运行时及时与您取得联系。”
```

Comment:
hahaha I just noticed this last sentence in the english version is not quite right

I'm changing it to say:
```
You will be able to provide your contact information as well as some basic hardware specs. It provides a way to open the lines of communication between you and the Moonbeam team in case any problems with your node arise.
```

Reply:
can you please update it here to reflect the new changes?